---
layout: ccsbtalk
title: "[CCSB Seminar Series] Deciphering cell-cell communications with spatial transcriptomic data"
speaker: Seungchan Kim
speakerurl: https://ccsb.pvamu.edu/team/seungchan-kim/
photo: /images/talks/speakers/seungchan-kim.png
date: 2022-10-05
time: 12:00pm
venue: webinar/Zoom
past-webinar: https://pvpanther.zoom.us/j/95093843065?pwd=bUprRllBVkRQUjBuNStuVFVNRDBTZz09
recording: https://pvpanther.zoom.us/rec/share/MqWm3fgNM3xw-NcLGhrou6T1MTGNgcobhMmG-h3rXQpbUc6VWq_TZXYP7uOnzUPE.2TOovAMavsunVKeU
categories: [blog, talks]
tags: [ccsb-seminar]
event: CCSB-Seminar
---


## Abstract

Cells do not live in isolation, and they send and receive messages from their environment and with neighboring cells or themselves, which is critical in cell's growth and normal operation.  For example, cell-cell communication allows specialization of groups of cells to form specific tissues such as muscle, blood, and brain cells, via intercellular coordination.  Disruption in cell-cell communication often leads to disease such as cancer.  

Recent advancement of single cell RNA sequencing (scRNAseq) is enabling scientists to study cellular and molecular heterogeneity of tissues and how such heterogeneities contributes to cell's fate.  In addition, there has been an increased interested in using scRNAseq data to study cell-cell communication by analyzing the expression of ligands and corresponding receptors.  Many computational tools and resources have been developed to support such analysis.  More recently, spatial transcriptomic (ST) technology has been developed to measure genome-wide gene expression while preserving the spatial information of cells.  Spatial information available from ST can help us decipher cell-cell communication in spatial contexts; however, none of the computational tools currently available takes this into account in analyzing ST data for cell-cell communication.  In this talk, I will introduce a computational approach that incorporates spatial distributions of cells in identifying active cell-cell communications and where cell-cell communications are most active.  Spatially resolved cell-cell communications can discover hotpots of cell signaling, directly overlaid on tissue, enabling biologists to associate cell's morphological features with cell signaling.



## Speaker Bio
Dr. Seungchan Kim is a Chief Scientist and Executive Professor at the [Department of Electrical and Computer Engineering](http://www.pvamu.edu/ece/) and the Director of the CRI [Center for Computational Systems Biology]({{site.baseurl}}{{"/"}}) at the [Prairie View A&M University (PVAMU)](http://www.pvamu.edu).  Prior to this appointment, he was the Head of Biocomputing Unit and an Associate Professor at Integrated Cancer Genomics Division of [Translational Genomics Research Institute (TGen)](http://www.tgen.org).  He was one of the founding faculty members of TGen, founded in 2002, by Dr. Trent, then-Scientific Director of the [National Human Genome Research Institute](https://www.genome.gov) at the [National Institutes of Health](https://www.nih.gov), leading computational systems biology research at the institute.  He was also an Assistant Professor in the [School of Computing, Informatics, Decision Systems Engineering (CIDSE)](https://cidse.engineering.asu.edu) at the [Arizona State University](http://www.asu.edu) from 2004 till 2011.  Dr. Kim received B.S. and M.S. degrees in [Agriculture Engineering](http://bse.snu.ac.kr/) from the [Seoul National University](http://www.snu.ac.kr), and Ph.D. in [Electrical Engineering](https://engineering.tamu.edu/electrical/) from the [Texas A&M University](http://www.tamu.edu). He also got his post-doctoral training at the Cancer Genetics Branch of [National Human Genome Research Institute](https://www.genome.gov).

Dr. Kim's research interests include: 1) mathematical modeling of genetic regulatory networks, 2) development of computational methods to analyze multitude of high throughput multi-omics data to identify disease biomarkers, and 3) computational models to diagnose patients or predict patient outcomes, for example, disease subtypes or drug response. His studies have had a large influence on the development of computational tools to study underlying mechanisms for cancer development and better understand the molecular mechanisms behind cancer biology and biological systems.



