---
layout: post
title: "Ms. Jalyn D Beaty successfully presented her MS project"
author: Seungchan Kim
image: 
categories: [blog]
tags: [news]
published: true
---


![Jalyn Beaty](/images/team/jalyn-beaty.png){: width="200" style="float: right; margin-left: 1em; margin-bottom:2em"}
On Wednesday, August 7, Ms. Jalyn D. Beaty successfully presented her MS project "Identifying Cell Types by Integrating Spatial Transcriptomics and Single Cell RNA Sequencing Data via k-NN Model".  She has completed BS/MS Program in Electrical and Computer Engineering Department in [Roy G. Perry College of Engineering](https://www.pvamu.edu/engineering/), [Prairie View A&M Universty](https://www.pvamu.edu) (PVAMU), and was a receipient of [I-ACED Scholarship](https://i-aced.org).  During her study at PVAMU, she also acquired work experience at Intel Corporation, Sandial National Laboratory and Lockheed Martin.  She is a member of Delta Sigma Theta Sorority Inc. and National Society of Black Engineers.  She currently works as a weapons integration engineer for Lockheed Martin.

The title, abstract and committee members are listed below:


<br />


**Title:** Identifying Cell Types by Integrating Spatial Transcriptomics and Single Cell RNA Sequencing Data via k- NN Model

**Abstract**
Understanding the cellular composition and spatial organization of tissues is essential for uncovering the complexities of biological systems and the diseases they are linked to. This research study introduces a new algorithm developed to combine single-cell RNA sequencing (scRNAseq) with spatial transcriptomic data, enhancing the ability to identify cell types and understand the layout of tissues. By using a data-driven method, this strategy seeks to improve the precision of identifying cell types and gain a more comprehensive understanding of tissue structure.

Single-cell RNA sequencing provides in-depth knowledge of the gene expression patterns of each cell, showing their distinct roles in the tissue. Spatial transcriptomics adds to this by aligning the expression of genes to their specific locations within the tissue, providing a spatial context to the gene expression information. However, the task of combining these datasets to accurately identify cell types is a challenging one.

This algorithm uses a set of reference data to train a K-Nearest Neighbors (k-NN) model, which is subsequently applied to the gene expression data from spatial transcriptomics to assign cell type labels. This process involves categorizing the data from single-cell RNA sequencing, training the k-NN model, and refining its performance by adjusting its parameters through repeated evaluation. The study applies this method to spatial data from different biological samples, including mouse muscle from the hindlimb, human breast cancer tissue, and mouse brain tissue from MCA205 tumor, to test the modelâ€™s effectiveness.

The findings show that by combining scRNA-seq and spatial transcriptomic data, the accuracy of identifying cell types is improved, and the grasp of the spatial variation within tissues is enhanced. This advance in methodologies has important implications for investigating the causes of diseases and developing targeted treatments, especially in the fields of cancer and other complex conditions. The algorithm proposed here marks an advancement in the field of spatial transcriptomics, offering a reliable tool for uncovering the complex relationship between cellular activity and spatial expression of genes


**MS Project Committee**

| **Role** | **Faculty** | **Title** |
| Chair: | Seungchan Kim, Ph.D. | Chief Scientist and Executive Professor, Electrical and Computer Engineering Department |
| Committee Member: | Suxia Cui, Ph.D., | Professor, Electrical and Computer Engineering Department |
| Committee Member: | Xishuang Dong, Ph.D. | Assistant Professor, Electrical and Computer Engineering Department |

